src/vtab/mod.rs

```diff
@@ -1063,12 +1044,6 @@ unsafe fn result_error<T>(ctx: *mut ffi::sqlite3_context, result: Result<T>) ->
     }
 }
 
-// Space to hold this string must be obtained
```

Comment:
`alloc` function moved to util module as it is now used by both vtab and error modules.